### ISSUE TITLE
Failing redirect to localhost & missing Docker volumes

### ISSUE THREAD

**Issue by ckamann** [smart-on-fhir/smart-dev-sandbox#5]
Hey,

I just set up the docker caontainer of smart and now I am facing the problem that on the "main page" of the alpine web server (http://10.12.255.223:4000/) all links refer to "localhost:port". This doesnt work because the docker instance is located at an own server and the links need to look like "http://10.12.255.223:port". Is there a possibility to customize this web page? I already modified the docker-compose.yml file and replaced all "localhosts" with the appropriate IP. And is it also normal that no volume is created in docker for the containers?

Regards from Germany
Christian

**Comment by vlad-ignatov** [comment]
You are right. This was created for local development and such an option was simply not predicted. Now that I think about it, having it running on another machine in the local network does make sense. I added the `HOST` variable to the `.env` file. If you set it to `HOST=10.12.255.223` and restart, it might solve your issue.

As for the volumes, I am not sure what you mean. Can you clarify?

**Comment by ckamann** [comment]
Thanks for your effort. I just downloaded the newest version and it works well. In case of the missing volumes I mean that there are no actual volumes that docker creates persistently for each of the containers. So usualy those are located in /var/lib/docker/volumes/ and also portainer displays them. Or is this not the intended behaviour?

**Comment by vlad-ignatov** [comment]
I am not familiar with portainer but I guess it lists the "global" volumes that you can create and then mount to any container... In this case, all the `dstu2-data*` and `dstu3-data*` folders are used as volumes so that you can persist your data outside of the container. They contain pre-populated databases  and are "mounted outside", so that you can preserve any changes.
